robots/pkg/kubevirt/cmd/get/presubmits.go

```diff
@@ -0,0 +1,151 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package get
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"html/template"
+	"kubevirt.io/project-infra/robots/pkg/kubevirt/log"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/test-infra/prow/config"
+	"kubevirt.io/project-infra/robots/pkg/kubevirt/cmd/flags"
+	"kubevirt.io/project-infra/robots/pkg/kubevirt/prowjobconfigs"
+)
+
+var getPresubmitsCommand = &cobra.Command{
+	Use:   "presubmits",
+	Short: "kubevirt get presubmits describes presubmit job definitions in project-infra for kubevirt/kubevirt repo",
+	RunE:  GetPresubmits,
+}
+
+func GetPresubmitsCommand() *cobra.Command {
+	return getPresubmitsCommand
+}
+
+type getPresubmitsJobruntimesOptions struct {
+	jobConfigPathKubevirtPresubmits string
+	outputFile                      string
+}
+
+func (o getPresubmitsJobruntimesOptions) Validate() error {
+	if _, err := os.Stat(o.jobConfigPathKubevirtPresubmits); os.IsNotExist(err) {
+		return fmt.Errorf("jobConfigPathKubevirtPeriodics is required: %v", err)
```

Comment:
```suggestion
		return fmt.Errorf("jobConfigPathKubevirtPresubmits is required: %v", err)
```

Reply:
THanks for spotting this. Updated, PTAL!